web/src/components/Trace/Trace.tsx

```diff
@@ -133,75 +131,36 @@ const Trace: React.FC<TraceProps> = ({testId, testResultId, onDismissTrace, onRu
     );
   }
 
+  const visiblePortion = 60;
+  const height = `calc(100% - ${visiblePortion}px)`;
```

Comment:
If these two constants are static, can we move them outside the component render section?